lib/_tls_wrap.js

```diff
@@ -918,7 +918,7 @@ Server.prototype.addContext = function(servername, context) {
 
   var re = new RegExp('^' +
                       servername.replace(/([\.^$+?\-\\[\]{}])/g, '\\$1')
-                                .replace(/\*/g, '[^\.]*') +
+                                .replace(/\*/g, '[^.]*') +
```

Comment:
This is only tangentially related to this PR, but was this line supposed to have a double backslash? This line looks like it was supposed to be

```js
.replace(/\*/g, '[^\\.]*')`
```

/cc @indutny based on git blame